core/src/main/java/greencity/repository/UserRepo.java

```diff
@@ -129,4 +129,15 @@
     @Transactional
     @Query(value = "UPDATE User SET last_activity_time=:userLastActivityTime WHERE id=:userId")
     void updateUserLastActivityTime(Long userId, Date userLastActivityTime);
+
+    /**
+     * Find the last activity time by {@link User}'s id.
+     *
+     * @param userId - {@link User}'s id
+     * @return {@link Date}
+     * @author Yurii Zhurakovskyi
+     */
+    @Query(nativeQuery = true,
```

Comment:
You can avoid @Query here and let spring-data handle it for you based on method name convention.